meteor/client/ui/DndListItemWrapper.tsx

```diff
@@ -0,0 +1,87 @@
+import { DragDropItemTypes } from './DragDropItemTypes'
+import React, { useRef } from 'react'
+import { DragSourceMonitor, DropTargetMonitor, useDrag, useDrop, XYCoord } from 'react-dnd'
+import { Identifier } from 'dnd-core'
+import { faGripLines } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+interface IDndItemWrapperProps {
```

Comment:
Don't use the `I` prefix. This isn't C#.
You are also inconsistent in using it. Only ten lines down there is another interface that doesn't have the prefix.

Reply:
Also, your component is called `DndListItemWrapper`, so this should be `DndListItemWrapperProps`.
If you leave out the `list` in the name, then you can't have list specific attributes like `index`

Reply:
I agree about the prefix, but this is the way that every single props and state interface has been written in the core. That's why i kept these the same to follow the structure, and any interface not intended for props and state do not use the prefix. 
The naming is simply a mistake and i will change it!

Reply:
I think i even mention this for you Ralle, that Lindved would comment on it. Didn't you write down a note about it, before christmas?

Reply:
I knew this would come up long before. I would like your opinions, however, on whether i should follow the conventions in the code in this specific scenario or just use the convention we usually adhere to even though it doesn't align with the other components.

Reply:
The prefixes have been changed with the newest commit.